package-lock.json

```diff
@@ -6229,6 +6230,11 @@
         "url": "https://github.com/sponsors/d-fischer"
       }
     },
+    "node_modules/cropperjs": {
+      "version": "1.6.2",
+      "resolved": "https://registry.npmjs.org/cropperjs/-/cropperjs-1.6.2.tgz",
+      "integrity": "sha512-nhymn9GdnV3CqiEHJVai54TULFAE3VshJTXSqSJKa8yXAKyBKDWdhHarnlIPrshJ0WMFTGuFvG02YjLXfPiuOA=="
+    },
```

Comment:
Cropper should be implemented like the other editors, without any additional installations of modules.